src/libraries/Microsoft.Extensions.Configuration.FileExtensions/src/FileConfigurationProvider.cs

```diff
@@ -118,6 +115,9 @@ static Stream OpenRead(IFileInfo fileInfo)
         /// </summary>
         /// <exception cref="FileNotFoundException">If Optional is <c>false</c> on the source and a
```

Comment:
Does it still throw a `FileNotFoundException` or is that going to be wrapped in the Inner exception?

Reply:
I corrected this, it was wrong to move the `OpenRead` call within the try / catch. Addressed in f3d2c4e